src/App.tsx

```diff
@@ -1,29 +1,70 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
+
+type Intial<T extends number[]> = {
+  total: number;
+  perPageOptions: T;
+  perPageInitial: T[number];
+};
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42).map(n => `Item ${n}`);
+
+const intialData: Intial<[3, 5, 10, 20]> = {
```

Comment:
There is a typo in the variable name 'intialData'. It should be 'initialData' to maintain proper naming conventions.